sensors/ev3_uart_sensor.h

```diff
@@ -18,6 +18,8 @@
 
 #include <lego_sensor_class.h>
 
+#define EV3_UART_SENSOR_NAME(suffix) "lego-ev3-uart-" suffix
```

Comment:
The suffix is actually the "type ID" of the sensor, so can we replace `suffix` with `type_id`?